android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/aptos/TestAptosSigner.kt

```diff
@@ -103,4 +103,47 @@ class TestAptosSigner {
             "07968dab936c1bad187c60ce4082f307d030d780e91e694ae03aef16aba73f3063000000000000000200000000000000000000000000000000000000000000000000000000000000010d6170746f735f6163636f756e74087472616e7366657200022007968dab936c1bad187c60ce4082f307d030d780e91e694ae03aef16aba73f3008e803000000000000fe4d3200000000006400000000000000c2276ada00000000210020ea526ba1710343d953461ff68641f1b7df5f23b9042ffa2d2a798d3adb3f3d6c405707246db31e2335edc4316a7a656a11691d1d1647f6e864d1ab12f43428aaaf806cf02120d0b608cdd89c5c904af7b137432aacdd60cc53f9fad7bd33578e01"
         )
     }
+
+    @Test
+    fun AptosTransferTokensCoins() {
+        // Successfully broadcasted https://explorer.aptoslabs.com/txn/0x60f7f7999bccac148c8b41b25097274e914216a10dc64a40a36c1f34e915ec2a?network=testnet
```

Comment:
Here it says "0 Amount"?

Reply:
Btw, I've put the link to a testnet transaction here, fixed
Thanks